Traceback (most recent call last):
  File "/home/jgualla1/vagueness/src/GQA_Data/Data_Sort_Small.py", line 40, in <module>
    random_num_list = random.sample(range(dictionary_list_len), slim_to)
  File "/home/jgualla1/anaconda3/lib/python3.7/random.py", line 321, in sample
    raise ValueError("Sample larger than population or is negative")
ValueError: Sample larger than population or is negative
